### ISSUE TITLE
Cannot use Fragment inside a Section

### ISSUE THREAD

**Issue by johngeorgewright** [yhatt/jsx-slack#319]
When using `</>` inside a `<Section/>` I get the following error.

```
<Section> has detected the unexpected component as an accessory: <Fragment>
```

Perhaps the [extra validation](https://github.com/yhatt/jsx-slack/blob/75b4bd27f891dc590c02896693ee5afae44010ae/src/block-kit/layout/Section.ts#L194-L211), in the `Section` component, needs to handle fragments?  Or is this limitation on purpose?

**Comment by yhatt** [comment]
Could you provide a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example)? jsx-slack has already should be supported the basic operation as like as including `<Fragment>` in `<Section>`. I could not reproduce the reported error in jsx-slack v6.

```jsx
import { Section, Fragment } from 'jsx-slack'

console.log(
  <Section>
    <p>Hello, world!</p>
    <Fragment>
      <p>foo</p>
      <p>bar</p>
    </Fragment>
  </Section>
)

/*
{
  type: 'section',
  text: {
    type: 'mrkdwn',
    text: 'Hello, world!\n\nfoo\n\nbar',
    verbatim: true
  }
}
*/
```

**Comment by johngeorgewright** [comment]
Ah, I have to use the import `Fragment`... I assumed that wouldn't be necessary. Here's my example:

```typescript
import { Section } from 'jsx-slack'

console.log(
  <Section>
    <p>Hello, world!</p>
    <>
      <p>foo</p>
      <p>bar</p>
    </>
  </Section>,
)
```

I can convert my project to use `Fragment` instead.

**Comment by yhatt** [comment]
`<Fragment>` and `<>` should use the exactly same Fragment component to work that correctly, but it looks like treated as different component:

```javascript
import { Fragment } from 'jsx-slack'

const fragment = <Fragment>a</Fragment>
const fragmentAlias = <>a</>

console.dir(fragment, { depth: null, showHidden: true })
console.dir(fragmentAlias, { depth: null, showHidden: true })

// 2 components should be same
console.log(fragment.$$jsxslack.type === fragmentAlias.$$jsxslack.type) // false
```

**Comment by yhatt** [comment]
Fixed in [v6.1.2](https://github.com/yhatt/jsx-slack/releases/tag/v6.1.2) 😎
